package.json

```diff
@@ -18,7 +18,7 @@
     "typings": "lib/src/index.d.ts",
     "scripts": {
         "clean": "rimraf lib",
-        "lint": "eslint --ext .ts,.json src/",
+        "lint": "eslint --fix --ext .ts,.json src/",

```

Comment:
The "--fix" option should only be used locally and any resulting changes should be checked in to git